tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/eval/ParseEvaluation.scala

```diff
@@ -0,0 +1,148 @@
+package org.allenai.nlpstack.parse.poly.eval
+
+import org.allenai.nlpstack.parse.poly.polyparser._
+
+object ParseEvaluation {
+
+  /** Runs a specified scoring function over all parses in the parse source, and sums the results.
+    *
+    * Each scoring function returns a ratio (i.e. numerator/denominator) for each parse in
+    * the parse source. The sum of these ratios is returned.
+    *
```

Comment:
Why do we need fraction represented as a tuple here rather than as a float or double value?